<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NTest-Creator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Налаштування тесту</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1000</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Час для проходження (хв)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditTitle"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Балів за кожну правильну відповідь</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Назва тесту</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Автор</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditAuthor"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Список студентів</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="text">
             <string>Додати список студентів</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOpen">
            <property name="text">
             <string>Редагувати список студентів</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBoxTime"/>
        </item>
        <item row="5" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxPoints"/>
        </item>
        <item row="8" column="0" colspan="4">
         <widget class="QPushButton" name="pushButtonSaveFirstPage">
          <property name="text">
           <string>Зберегти</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Редактор запитань</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="8" column="1" rowspan="2" colspan="6">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonSaveNewTest">
            <property name="text">
             <string>Зберегти</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonAddQ">
              <property name="text">
               <string>Додати запитання</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDeleteQ">
              <property name="text">
               <string>Видалити запитання</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="pushButtonPrevious">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QLineEdit" name="lineEditQuestion">
          <property name="placeholderText">
           <string>Текст запитання</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Тип запитання</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QComboBox" name="comboBoxType"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEditNote">
          <property name="placeholderText">
           <string>Примітка</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditAnswer1">
          <property name="placeholderText">
           <string>Варіант відповіді</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLineEdit" name="lineEditAnswer3">
          <property name="placeholderText">
           <string>Варіант відповіді</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEditAnswer2">
          <property name="placeholderText">
           <string>Варіант відповіді</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QLineEdit" name="lineEditAnswer4">
          <property name="placeholderText">
           <string>Варіант відповіді</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QLabel" name="label_7">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Правильна відповідь</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="5" colspan="2">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>100</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioButton_3">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>в</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_2">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>б</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="radioButton_4">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>г</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>а</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>в</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>б</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>г</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <widget class="QLineEdit" name="lineEditCorrectAnswer">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="pushButtonNext">
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>г</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>в</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>б</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>a</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreateTest">
   <property name="text">
    <string>Створити</string>
   </property>
  </action>
  <action name="actionEditTest">
   <property name="text">
    <string>Редагувати</string>
   </property>
  </action>
  <action name="actionCreateList">
   <property name="text">
    <string>Створити список</string>
   </property>
  </action>
  <action name="actionOpenList">
   <property name="text">
    <string>Редагувати існуючий</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>lineEditAuthor</tabstop>
  <tabstop>spinBoxTime</tabstop>
  <tabstop>doubleSpinBoxPoints</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonOpen</tabstop>
  <tabstop>pushButtonSaveFirstPage</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>lineEditQuestion</tabstop>
  <tabstop>lineEditNote</tabstop>
  <tabstop>lineEditAnswer1</tabstop>
  <tabstop>lineEditAnswer2</tabstop>
  <tabstop>lineEditAnswer3</tabstop>
  <tabstop>lineEditAnswer4</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>lineEditCorrectAnswer</tabstop>
  <tabstop>pushButtonSaveNewTest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
